api/openapi-spec/swagger.json

```diff
@@ -17898,6 +17926,20 @@
      }
     }
    },
+   "v1.SeccompConfiguration": {
+    "type": "object",
+    "required": [
+     "installPolicy"
+    ],
+    "properties": {
+     "installPolicy": {
```

Comment:
Can we have a doc string for this?

Reply:
Maybe we should call this `installKubeVirtSeccompDefaultProfile`?

Reply:
+1 We definitely need doc string for the new API